herbridge/frontend/src/components/App/App.js

```diff
@@ -171,7 +174,12 @@ class App extends React.Component {
     const newSelectedPhotos =
       flatMap(imageSections, (section, sectionIndex) => {
         return section.images.filter((image, imageIndex) => {
-          return indexes[sectionIndex].includes(imageIndex)
+          const indexesForSection = indexes[sectionIndex]
+          if (indexesForSection !== undefined) {
+            return indexesForSection.includes(imageIndex)
+          } else {
+            return false
+          }
```

Comment:
This could be `return indexForSection !== undefined && indexesForSection.includes(imageIndex)` if you want to go for max terseness